test/e2e/config/azure-dev.yaml

```diff
@@ -206,14 +206,14 @@ providers:
 variables:
   AKS_KUBERNETES_VERSION: "latest"
   AKS_KUBERNETES_VERSION_UPGRADE_FROM: "latest-1"
-  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.29}"
+  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.32}"
   FLATCAR_KUBERNETES_VERSION: "${FLATCAR_KUBERNETES_VERSION:-stable-1.26}"
-  KUBERNETES_VERSION_API_UPGRADE_FROM: "v1.28.10"
+  KUBERNETES_VERSION_API_UPGRADE_FROM: "v1.31.9"
   FLATCAR_VERSION: "${FLATCAR_VERSION:-latest}"
-  ETCD_VERSION_UPGRADE_TO: "3.5.10-0"
-  COREDNS_VERSION_UPGRADE_TO: "v1.11.1"
-  KUBERNETES_VERSION_UPGRADE_TO: "${KUBERNETES_VERSION_UPGRADE_TO:-stable-1.29}"
-  KUBERNETES_VERSION_UPGRADE_FROM: "${KUBERNETES_VERSION_UPGRADE_FROM:-stable-1.28}"
+  ETCD_VERSION_UPGRADE_TO: "" # Use default
+  COREDNS_VERSION_UPGRADE_TO: "" # Use default
```

Comment:
FYI I extracted this into its own PR so we can get this into release branches which I see are hitting issues with these versions: #5677